src/main/java/com/codicesoftware/plugins/hudson/model/ChangeSet.java

```diff
@@ -112,6 +149,7 @@ public void setRepoServer(String repoServer) {
     }
 
     @Exported
+    @Deprecated
```

Comment:
Question: why is this getter deprecated now?

Reply:
I've marked it as deprecated because it returned server with protocol and port number that made Object Specs harder to read. Views does not use it anymore. Instead they reference server and repository name fields directly. Removing annotation.

Reply:
Ok, you can keep the anotation then! I just wanted to know why, yours seems like a pretty good reason.